### PR TITLE
Add location to create event

### LINKED ISSUE #122: Location and submission time for create section event
Use a library for time picker and gmaps (or osm) for location

### PR DESCRIPTION
**Pull request by kmcnellis**
fixes #122

### CODE REVIEW

File: client/app/app.js [hunk 1]
```diff
@@ -23,4 +24,16 @@ angular.module('venueApp', [
       });
 
     $locationProvider.html5Mode(true);
-  });
+  })
+  .config(function(uiGmapGoogleMapApiProvider) {
+    uiGmapGoogleMapApiProvider.configure({
+        //    key: 'your api key',
```
Comment (seveibar) [inline client/app/app.js:30]: this needs to be in some config file- i know we don't really have a config file for the client but this should be in it- we could use angular.constant

File: client/components/eventform/eventform.controller.js [hunk 2]
```diff
@@ -12,6 +12,145 @@ angular.module('venueApp')
...
+
+    $scope.event.searchbox= {
+      template: 'searchbox.tpl.html',
+      //position:'top-right',
+      position:'top-left',
+      options: {
+        bounds: {},
+        visible: true
+      },
+      //parentdiv:'searchBoxParent',
```
Comment (seveibar) [inline client/components/eventform/eventform.controller.js]: ghost code

File: client/components/eventform/eventform.controller.js [hunk 3]
```diff
@@ -12,6 +12,145 @@ angular.module('venueApp')
...
+          $scope.event.searchbox.options.bounds = new google.maps.LatLngBounds(sw, ne);
+          //$scope.event.searchbox.options.visible = true;
+        }
+      }
+    };
+    var places = [];
+
+    $scope.event.searchbox= {
+      template: 'searchbox.tpl.html',
+      //position:'top-right',
```
Comment (seveibar) [inline client/components/eventform/eventform.controller.js]: ghost code

File: client/components/eventform/eventform.controller.js [hunk 4]
```diff
@@ -12,6 +12,145 @@ angular.module('venueApp')
...
+        places_changed: function (searchBox) {
+
+          if (searchBox){
+            places = searchBox.getPlaces()
+          }
+
+          if (places.length == 0) {
+            return;
+          }
+          // For each place, get the icon, place name, and location.
```
Comment (seveibar) [inline client/components/eventform/eventform.controller.js:99]: good comment

File: client/app/app.js (hunk 1 above)
Comment (kmcnellis) [inline client/app/app.js:30]: Not really sure that we need an api key - seems to work fine without it

**Comment by jowlee** [comment]
Looks great!

**Pull request merged**